app/services/proofing/resolution/progressive_proofer.rb

```diff
@@ -13,12 +13,8 @@ def initialize(instant_verify_ab_test_discriminator = nil)
       end
 
       # @param [Hash] applicant_pii keys are symbols and values are strings, confidential user info
-      # @param [Boolean] double_address_verification flag that indicates if user will have
-      #   both state id address and current residential address verified. Note this value is here as
-      #   a placeholder until it can be replaced with ipp_enrollment_in_progress in ticket LG-353:
-      #   https://cm-jira.usa.gov/browse/LG-11353
-      # @param [Boolean] ipp_enrollment_in_progress flag that indicates if user will have
```

Comment:
we should probably still keep this definition in addition to the newly added info that it's used in place of the dav flag

Reply:
Is the DAV flag still in place? I thought it was removed from the config?

Reply:
it's removed - i was thinking the new comment should be more like: ~ `# @param [Boolean] ipp_enrollment_in_progress flag is used in place of DAV flag because DAV is
      #   now always true
# it indicates if both state id address and current residential address verified`

Reply:
I thought ipp_enrollment_in_progress indicates that an enrollment is either in the 'establishing' or 'in progress' statuses? Happy to be wrong though!

Reply:
yes but it was also [meant as a replacement](https://cm-jira.usa.gov/browse/LG-11235) for `double_address_verification` . So it's like an in person enrollment exists for the user acting as a stand in for the user is going through the ipp flow.